app/js/utils/DateTime.js

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
I grabbed the LogDecorator from a tutorial I found. I think moment.js is a bit much for what the DateTime is used for in the LogDecorator. If it was part of the UI or essential business logic I would add moment.js as a dependency.